### PR TITLE
Provide API/appslist

### PR DESCRIPTION
**Pull request by lslachciak**
[Issue] https://github.com/Samsung/TAU/issues/1419
[Problem] Need to extend current API/webcliplist
[Solution] Replace old API with new one containing apps list.

Signed-off-by: Lukasz Slachciak <l.slachciak@samsung.com>

### CODE REVIEW

File: examples/mobile/HomeApp/js/app.js [hunk 1]
```diff
@@ -6,60 +6,100 @@ import Storage from "./clipping-storage.js";
...
-					[
-						"webclip/tv-remote-control",
-						"webclip/restaurant",
-						"webclip/weather",
-						"webclip/apps-on-tv",
-						"webclip/latest-news",
-						"webclip/now-on-tv"
-					];
+				// use default webclip list if sth wrong
+				appsList = [
```
Comment (TomaszLukawskiSam) [inline examples/mobile/HomeApp/js/app.js]: Please, move this object to some object variable for module like defaultList.

Comment (TomaszLukawskiSam) [reply examples/mobile/HomeApp/js/app.js]: The structure of this object seems appropriate to me. We can change it during the project development.

Comment (lslachciak) [reply examples/mobile/HomeApp/js/app.js]: done

File: examples/mobile/HomeApp/js/app.js [hunk 2]
```diff
@@ -171,7 +218,16 @@ import Storage from "./clipping-storage.js";
 		burgerButton.addEventListener("click", onButtonClick);
 		popupButton.addEventListener("click", onPopupSubmit);
 
-		loadWebClipList();
+		appsList = storage.readAllFromStorage(Storage.elements.APPSLIST);
```
Comment (TomaszLukawskiSam) [inline examples/mobile/HomeApp/js/app.js]: You can add default value for `appsList` eg.;
```
appsList = storage.readAllFromStorage(Storage.elements.APPSLIST) || [];
```

File: examples/mobile/HomeApp/js/app.js [hunk 3]
```diff
@@ -171,7 +218,16 @@ import Storage from "./clipping-storage.js";
...
+
+		//TODO: all webclips url should checked if they are valid
+
+		if (!Array.isArray(appsList) || !appsList.length) {
```
Comment (TomaszLukawskiSam) [inline examples/mobile/HomeApp/js/app.js]: And remove condition `!Array.isArray(appsList)`

File: examples/mobile/HomeApp/js/app.js (hunk 2 above)
Comment (lslachciak) [reply examples/mobile/HomeApp/js/app.js]: done

File: examples/mobile/HomeApp/js/app.js (hunk 3 above)
Comment (lslachciak) [reply examples/mobile/HomeApp/js/app.js]: done

**Comment by lslachciak** [comment]
improved according to @TomaszLukawskiSam comments

**Review by TomaszLukawskiSam — APPROVED** [review]
Looks good to me.

**Pull request merged**